src/bundlers/utils.ts

```diff
@@ -0,0 +1,81 @@
+import log from "loglevel";
+import { ClassConfiguration } from "../models/projectConfiguration.js";
+import { fileExists } from "../utils/file.js";
+import { BundlerInterface, BundlerOutput } from "./bundler.interface.js";
+import { TsRequiredDepsBundler } from "./node/typescriptRequiredDepsBundler.js";
+import { TypeCheckerBundler } from "./node/typeCheckerBundler.js";
+import { NodeJsBundler } from "./node/nodeJsBundler.js";
+import { NodeJsBinaryDependenciesBundler } from "./node/nodeJsBinaryDependenciesBundler.js";
+import { BundlerComposer } from "./bundlerComposer.js";
+import { DartBundler } from "./dart/dartBundler.js";
+import { KotlinBundler } from "./kotlin/kotlinBundler.js";
+import { debugLogger } from "../utils/logging.js";
+import { createTemporaryFolder } from "../utils/file.js";
+import { ProjectConfiguration } from "../models/projectConfiguration.js";
+import { Program } from "../models/genezioModels.js";
+
+export async function bundle(
```

Comment:
I have seen that this code is duplicated twice in `deploy.ts` line 289 and in `local.ts` line 508 (with small differences).

Can you please only write it here in `utils.ts` and reuse it everywhere (3 places). It would help if we want to change the bundling system in the future, because we would not need to change the code in 3 places.